HACKTOBERFEST.md

```diff
@@ -0,0 +1,57 @@
+# Appwrite hacktoberfest contribution
+
+Welcome to the Appwrite Hacktoberfest contribution! We appreciate your interest in contributing to our open-source project. Please read this carefully to understand how to get started and make your contributions count.
+
+## Familiarize with the project
+
+- Before you start contributing, it's essential to get familiarized with our project. You can do that by following the [main README](https://github.com/appwrite/appwrite/blob/main/README.md).
```

Comment:
It doesn't seem like this needs to be a bullet. Also, some Grammarly suggestions

```suggestion
Before you start contributing, familiarize yourself with our project by reading the [main README](https://github.com/appwrite/appwrite/blob/main/README.md).
```